var/spack/repos/builtin/packages/kokkos/package.py

```diff
@@ -408,39 +406,35 @@ def setup_build_tests(self):
         self.cache_extra_test_sources(cmake_out_path)
         self.cmake_bin(set=True)
 
-    def build_tests(self, cmake_path):
+    def test_build(self):
```

Comment:
This isn't a test part; rather it should stay `build_tests()` but with a replacement of the cmake hack..

Reply:
Actually, I would get rid of this method altogether, moving the call to `cmake` and `make` into the test method.  The test should be cleaner without the `cmake_bin` hack.